Prototypes/MorganStaneley.ComposeUI.GridSelectionPrototype/.gitignore

```diff
@@ -0,0 +1,454 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
There should be a single `.gitignore` file at the root of the repo. @nick-polyak-ms is there a reason why you're adding these fileS?

Reply:
Yes, I already responded to mgasca promising to have a single .gitignore.

Reply:
need help merging all .gitignore files into a single one that would work for every solution.

Reply:
merged manually